README.md

```diff
@@ -123,6 +123,13 @@ let width = self.usernameLabel.width()
 
 ## UIView
 
+### Nib support
+
+Helper class method to create a view from a nib file directly. Supports overriding the bundle and the name of the nib.
+```swift
+let myView = MyView.instanceFromNib(nibName: nil, bundle:  .main)
```

Comment:
if `nil` - better to not pass it right ?

Reply:
Yeah, just passed it in the docs to better exemplify usage. 👍

Reply:
then better to pass some string ;)

Reply:
Done. 🚀